### PR TITLE
SAP HANA Cloud Getting Started tutorials

### PR DESCRIPTION
**Pull request by tracywai-sap**
Updated and added a new tutorial to the Jump Start mission

### CODE REVIEW

File: tutorials/hana-cloud-mission-trial-9/hana-cloud-mission-trial-9.md [hunk 1]
```diff
@@ -3,26 +3,27 @@ parser: v2
 author_name: Christopher Kollhed
 author_profile: https://github.com/chriskollhed
 auto_validation: true
-time: 10
+time: 15
 tags: [ tutorial>beginner, products>sap-hana-cloud, software-product-function>sap-hana-cloud\,-sap-hana-database, products>sap-business-application-studio]
```
Comment (jmmargo) [inline tutorials/hana-cloud-mission-trial-9/hana-cloud-mission-trial-9.md:7]: "products" should be "software-product" in all tags.

File: tutorials/hana-cloud-mission-trial-9/hana-cloud-mission-trial-9.md [hunk 2]
```diff
@@ -3,26 +3,27 @@ parser: v2
...
 primary_tag: products>sap-hana-cloud
```
Comment (jmmargo) [inline tutorials/hana-cloud-mission-trial-9/hana-cloud-mission-trial-9.md:8]: Please add "parser: v2" as the last item in the meta data section, below the primary tag.

File: tutorials/hana-cloud-mission-trial-9/hana-cloud-mission-trial-9.md [hunk 3]
```diff
@@ -33,210 +34,259 @@ primary_tag: products>sap-hana-cloud
...
+In this tutorial, you will learn how to create a calculation in SAP Business Application Studio, in which you will join tables and rank results to get Alex the business insights they need.
 
 ---
 
-### Generate a SELECT statement on the column view
+### Create the calculation view
 
 
-To allow others to see the results of your calculation view, you need to grant them the privilege to run `SELECT` statements on this calculation view.
+1.	Within your project in the SAP Business Application Studio, click on **View** from the side menu. Then click on **Command Palette**. Alternatively, use `Ctrl+Shift+P` to access it.
```
Comment (jmmargo) [inline tutorials/hana-cloud-mission-trial-9/hana-cloud-mission-trial-9.md:44]: The extra spaces in numbers 1 and 2 are not consistent with 3 and 4.

File: tutorials/hana-cloud-mission-trial-8/hana-cloud-mission-trial-8.md [hunk 4]
```diff
@@ -8,22 +8,23 @@ tags: [ tutorial>beginner, products>sap-hana-cloud, software-product-function>sa
 primary_tag: products>sap-hana-cloud
```
Comment (jmmargo) [inline tutorials/hana-cloud-mission-trial-8/hana-cloud-mission-trial-8.md:8]: "products" should be "software-product"

File: tutorials/hana-cloud-mission-trial-7/hana-cloud-mission-trial-7.md [hunk 5]
```diff
@@ -3,27 +3,26 @@ parser: v2
 author_name: Christopher Kollhed
 author_profile: https://github.com/chriskollhed
 auto_validation: true
-time: 15
-tags: [ tutorial>beginner, products>sap-hana-cloud, software-product-function>sap-hana-cloud\,-sap-hana-database, products>sap-business-application-studio]
+time: 10
+tags: [ tutorial>beginner, products>sap-hana-cloud, software-product-function>sap-hana-cloud\,-sap-hana-database, programming-tool>sql]
 primary_tag: products>sap-hana-cloud
```
Comment (jmmargo) [inline tutorials/hana-cloud-mission-trial-7/hana-cloud-mission-trial-7.md:8]: "products" should be "software-product"

File: tutorials/hana-cloud-mission-trial-6/hana-cloud-mission-trial-6.md [hunk 6]
```diff
@@ -3,26 +3,26 @@ parser: v2
 author_name: Christopher Kollhed
 author_profile: https://github.com/chriskollhed
 auto_validation: true
-time: 10
-tags: [ tutorial>beginner, products>sap-hana-cloud, software-product-function>sap-hana-cloud\,-sap-hana-database, programming-tool>sql]
+time: 5
+tags: [ tutorial>beginner, products>sap-hana-cloud, software-product-function>sap-hana-cloud\,-sap-hana-database]
 primary_tag: products>sap-hana-cloud
```
Comment (jmmargo) [inline tutorials/hana-cloud-mission-trial-6/hana-cloud-mission-trial-6.md:8]: "products" should be "software-product"

**Review by jmmargo — CHANGES_REQUESTED** [review]

**Comment by jmmargo** [comment]
Tracy, please be sure that in every location "products" is used in the tags or primary tag, "software-product" is used in its place.

**Review by tracywai-sap — COMMENTED** [review]
fixed tags

**Pull request merged**